gradle/wrapper/gradle-wrapper.properties

```diff
@@ -11,6 +11,6 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
```

Comment:
OpenSearch is using 8.1.1: https://github.com/opensearch-project/OpenSearch/blob/2.x/gradle/wrapper/gradle-wrapper.properties
Shouldnt CCR be in sync with OpenSearch ?